src/Module.Server/ModuleData/dtv/dtv_class_divisions.xml

```diff
@@ -1870,6 +1870,270 @@
       </Perk>
     </Perks>
   </MPClassDivision>
+  
+ <MPClassDivision id="crpg_dtv_greyorder_shielder"
+                   hero="crpg_dtv_greyorder_shielder"
+                   troop="crpg_dtv_greyorder_shielder"
+                   hero_idle_anim="act_vlandia_mp_voulgier_idle"
+                   troop_idle_anim="act_idle_spear_1_left_stance"
+                   multiplier="0"
+                   cost="0"
+                   icon="Special_Swordsman"
+                   melee_ai="50"
+                   ranged_ai="0"
+                   armor="0"
+                   movement_speed="0"
+                   combat_movement_speed="0"
+                   acceleration="0">
+    <Perks>
+      <Perk game_mode="all"
+            name="{=}None"
+            description="{=}None."
+            icon="PerkToughness"
+            perk_list="1">
+      </Perk>
+    </Perks>
+  </MPClassDivision>
+  
+  <MPClassDivision id="crpg_dtv_greyorder_kuyak_two_handed"
+                   hero="crpg_dtv_greyorder_kuyak_two_handed"
+                   troop="crpg_dtv_greyorder_kuyak_two_handed"
+                   hero_idle_anim="act_vlandia_mp_voulgier_idle"
+                   troop_idle_anim="act_idle_spear_1_left_stance"
+                   multiplier="0"
+                   cost="0"
+                   icon="Special_Swordsman"
+                   melee_ai="70"
```

Comment:
I would bring the `melee_ai` down for all of your regular units. Tincans & dwarves have only 30 in comparison to your 70. Maybe make them 50 so they are still formidable in melee to account for the lack of ranged units.